cmds/core/init/init_linux.go

```diff
@@ -6,17 +6,57 @@ package main
 
 import (
 	"log"
+	"os"
+	"path/filepath"
 	"strconv"
+	"strings"
 	"syscall"
 
 	"github.com/u-root/u-root/pkg/cmdline"
+	"github.com/u-root/u-root/pkg/kmodule"
 )
 
+// installModules installs kernel modules (.ko files) from /lib/modules.
+// Useful for modules that need to be loaded for boot (ie a network
+// driver needed for netboot)
+func installModules() {
+	modulePattern := "/lib/modules/*.ko"
+	files, err := filepath.Glob(modulePattern)
+	if err != nil {
+		log.Printf("installModules: error finding files at %q: %v", modulePattern, err)
```

Comment:
you'll get an error if the glob is bad, so maybe you want to just print the return from Glob

Reply:
Fixed in latest.